safe/definitions/hazard.py

```diff
@@ -231,7 +232,59 @@
     'layer_modes': [layer_mode_classified, layer_mode_continuous],
     'disabled_exposures': [exposure_place]
 }
+hazard_dam_break = {
+    'key': 'dam_break',
+    'name': tr('DAM Break'),
```

Comment:
I'm wondering if we should make it lower case. None of our hazard are written with capital letters

Reply:
Thanks, i have something to asked but then i was confusing by other project so i forgot to ask those.. 
Actually i've been removed those in my local repo.. but if you could help me to explain what is that dictionary suppose to be? should i put that in my hazard classification?

Reply:
hhm what about this?
 hazard_flood = {
    'key': 'flood',
    'name': tr('Flood')

Reply:
oh i see..

Reply:
`name` is the human friendly name, while `key` is the computer friendly one.
Does `DAM` means `Bendungan` in Bahasa? If so, you should write it as `Dam`. Or is it a abbreviation?

I think you should also explain what is DAM in this case in the description.